pontoon/base/views.py

```diff
@@ -380,21 +379,20 @@ def get_translations_from_other_locales(request):
 
 @utils.require_AJAX
 # need to define preferred_source_locale
-def get_sibling_entities(request): 
+def get_sibling_entities(request):
     """Get entities preceding and succeeding the current entity"""
     print("TEST-------------------")
     try:
         entity = int(request.GET["entity"])
         locale = request.GET["locale"]
-        print(entity, locale)
     except (MultiValueDictKeyError, ValueError) as e:
         return JsonResponse(
             {"status": False, "message": "Bad Request: {error}".format(error=e)},
             status=400,
         )
 
     entities = Entity.objects.get(id=entity)
```

Comment:
can you show me the magic to make this more DRY. I can't figure out how to use the QuerySets properly

Reply:
1. Remove this line. :)
2. As you can see below, `preceding_entities` and `succeeding_entities` QuerySets share the first two lines, so you can factor them out into an `entities` QuerySet and reuse it, e.g. `preceding = entities.filter(...)`.
3. Also, move both `get_object_or_404` declarations after the `try - except` block, because you'll need `entity` in `entity.resource` and `entity.order`.

Reply:
This makes more sense after the question you answered on Element.
I didn't realize that `entity` and `locale` were getting the model object when using the `get_object_or_404` method
but hence the name of the method.  so it came with all the models plus the one that is automatically added the `id` which I was looking for so I can filter out the entities in the first place :)